docs/main.md

```diff
@@ -0,0 +1,220 @@
+# airwallex-payment-elements Documentation
+
+See `node_modules/airwallex-payment-elements/types` for a more detailed overview of the props available for each function.
+
+Functions:
+
+- [loadAirwallex](#loadAirwallex)
+- [redirectToCheckout](#redirectToCheckout)
+- [confirmPaymentIntent](#confirmPaymentIntent)
+- [confirmPaymentIntentWithSavedCard](#confirmPaymentIntentWithSavedCard)
+- [createPaymentMethod](#createPaymentMethod)
+- [getPaymentIntent](#getPaymentIntent)
+- [createPaymentContract](#createPaymentContract)
+
+- [Element](#Element)
+- [createElement](#createElement)
+- [destroyElement](#destroyElement)
+- [getElement](#getElement)
+- [setElement](#setElement)
```

Comment:
I am thinking of group them in three sections:

initialization

Interact with element

Recurring choice

Confirm and make payment

Query API